openpype/hosts/fusion/api/menu.py

```diff
@@ -1,26 +1,21 @@
-import os
 import sys
 
 from Qt import QtWidgets, QtCore
 
-from openpype import style
+from avalon import api
```

Comment:
Wow. This is such old PR?

Reply:
Definitely is - I'll test it soon and clean up all the bits :D